docker-compose.yml

```diff
@@ -372,6 +372,26 @@ services:
          - opentsdb.pendev
 
 
+  openkilda-gui:
+    container_name: openkilda-gui
+    build:
+      context: services/src/openkilda-gui
+    image: "kilda/openkilda-gui:${full_build_number:-latest}"
+    ports:
+      - "1010:1010"
```

Comment:
Usage of port number smaller than 1024 is a bad idea.

Reply:
This port is used from the starting of this project and if we change this now we have to notify users who all are using kilda gui.
Ideally when using GUI port should not be visible.

Reply:
I do not ask to change "production" TCP port, I am asking to not use ports numbers smaller than 1024 in dev/local env.

Reply:
Ok. For GUI it is better to run on 80 port so that no one need to add additional port information while opening the GUI. 
Port 80 was consumed by TPE service, once TPE shift to another port will change 1010 to 80.

Reply:
One more time - I din't suggest to change "production application port", I have asked to change "development port" and do not use reserved/common used port for it.